### DISCUSSION TITLE
Just got the new Standard Notes pricing structure email. Does this mean that the free plan is gouing away?

### DISCUSSION THREAD

**Discussion by Superpigdots** [standardnotes/mobile#503] [Q&A] [answered]
I just got that long email today about how the subscription plans are changing in their pricing structure. For those of us who use the app in a minimal capacity, I would like to know: is the free plan is going away? The email did not even seem to mention that.

**Comment by atmoio** [comment, marked as answer]
Nope, free plan stays exactly the same! Indeed, in all the excitement about the changes we forgot to reassure free users that nothing will change for them :)

**Reply by Superpigdots** [reply]
Sweet, thank you for the info.
